src/odemis/driver/xt_client.py

```diff
@@ -1447,6 +1458,24 @@ def _cancelAutoContrastBrightness(self, future):
     #         return True
 
 
+class FibScanner(model.Emitter):
+    """
+    This is an extension of the model.Emitter class for controlling the FIB. Currently via XTLib only minimal control
+    of the FIB is possible, which is limited to pause/unpausing the image beam, and getting the latest image (via the
+    detector with the appropriate channel).
+    """
+
+    def __init__(self, name, role, parent, **kwargs):
+        model.Emitter.__init__(self, name, role, parent=parent, **kwargs)
+
+        # Set unknown values to None or 0
+        self.dwellTime = model.VigilantAttribute(None, readonly=True)
+        self._shape = (0,)
```

Comment:
We don't know the shape, but something like 4096,4096 would be more reasonable.
We might want to have a read-only .resolution VA with the same values. There is a lot of code that expects to get this info.